pds4indextools/default_values.yaml

```diff
@@ -0,0 +1,21 @@
+{'version_id': '1.1',
```

Comment:
While JSON is generally parsable by YAML readers, the main advantage of YAML is it's inline format, which is easier to write and read by humans. You might consider changing this to:
```
- version_id: '1.1'
- title: 'Tested Index File'
```

Also there are a lot of blank lines at the bottom of the file.